app/controllers/projects_controller.rb

```diff
@@ -37,6 +37,24 @@ def update
     end
   end
 
+  def destroy
+    @project = Project.find_by(id: params[:id])
+
+    if policy(@project).valid_deactivated_at?(DateTime.now)
```

Comment:
Itt olyan érzésem van, mintha a `valid_deactivated_at?` -re szeretnéd bízni, hogy a felhasználó tudja-e deaktiválni a projektet.

Eddig ugye a sima policy-kat arra használtuk, hogy "az adott erőforráshoz hozzáférhet-e a felhasználó?", és az attribute_policy-kat arra, hogy "az egyes attribútumokat módosíthatja-e valamilyen értékre a felhasználó?".
- Ez alapján érdemes lenne az authorizációt továbbra is a sima policy-ra bízni (pl az egyel fentebbi kommentem). 
- Szerintem attribute_policy-ra itt nincs szükségünk.
  - Felhasználófüggetlenül mindig ugyanazt az értéket fogjuk beállítani (`DateTime.now`).
  - Ugyanazok a felhasználók állíthatják be azt az értéket, mint akik a többi attribútumát is tudják módosítani a Project-nek.